### ISSUE TITLE
Please support minification

### ISSUE THREAD

**Issue by mcasimir** [cgarvis/angular-toggle-switch#23]
Minification with your code breaks everytihing. Solution: use this:

``` js
    controller: ['$scope', function($scope){
      $scope.toggle = function toggle() {
        if(!$scope.disabled) {
          $scope.model = !$scope.model;
        }
      };
    }],
```

Instead of:

``` js
    controller: function($scope) {
      $scope.toggle = function toggle() {
        if(!$scope.disabled) {
          $scope.model = !$scope.model;
        }
      };
    },
```

Thank You!

**Comment by adamdicarlo** [comment]
I'd rather my own build do the minification (and have the option to generate a source map, which is not included in angular-toggle-switch). Mostly because then I don't have to switch out which version of the library I use for debug builds and release builds. (My build system doesn't have ngmin and it'd be a pain to add, unfortunately.)

I figured since ngmin is set up in your build you wouldn't want to add the annotation, though, so I just added it myself in my own fork but excluded that from the PR I submitted. ;)
